nomt/src/beatree/ops/update/leaf_updater.rs

```diff
@@ -14,17 +14,61 @@ use super::{
 
 pub struct BaseLeaf {
     pub node: LeafNode,
-    pub iter_pos: usize,
     pub separator: Key,
+    low: usize,
```

Comment:
might as well just make the `BaseLeaf` fields all private at this point (has needed doing for some time!)